.github/workflows/ci.yml

```
@@ -50,6 +50,23 @@ jobs:
       - name: cargo test (ignored)
         run: cargo test -p pest_grammars --lib --verbose --release -- --ignored tests::toml_handles_deep_nesting_unstable
 
+  check-grammar-sync:
+    name: Ensure grammar.rs matches grammar.pest
+    needs: check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)